pkg/config/config.go

```diff
@@ -390,3 +392,14 @@ func isConfigProvided(cliArgName string, envVarName string) bool {
 	})
 	return cliArgProvided
 }
+
+func getRegionFromQueueURL(queueURL string) string {
+	for _, partition := range endpoints.DefaultPartitions() {
+		for regionID := range partition.Regions() {
+			if strings.Contains(queueURL, regionID) {
```

Comment:
it's not likely; but if someone named their queue `us-west-2` when it was actually in e.g. `ap-southeast-1` then this could have a false positive.